### PR TITLE
Add blog post and cleanup usage page.

### PR DESCRIPTION
**Pull request by rgwozdz**

### CODE REVIEW

File: _docs/usage/provider.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ koop.register(socrata, {
 ```
 <figcaption><i>Figure 3. Example of using a <code class='highlighter-rouge'>before</code> function to reject specific requests.</i></figcaption>
 
-The before function can also be used to modify the `request` objects.  In the example below, query parameter `limit` is assigned a value of 500 before it gets used by the provider's `getData` function.
+The before function can also be used to modify the `request` objects.  In the example below, query parameter `limit` is assigned a value of 500 before it gets used by the provider's `getData` function. In the example below, we use the `before` function, to replace the `outSR` parameter is with its WKT equivalent; this allows Koop to successfully reproject the data to the desired output spatial reference:
```
Comment (haoliangyu) [inline _docs/usage/provider.md]: You should remove this line:

```
In the example below, query parameter `limit` is assigned a value of 500 before it gets used by the provider's `getData` function.
```

File: _posts/2020-03-09-proivder-transformation-functions.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title:  "Provider transformation functions"
```
Comment (haoliangyu) [inline _posts/2020-03-09-proivder-transformation-functions.md]: Maybe change to `Introduction to provider transformation functions`

Comment (rgwozdz) [reply _posts/2020-03-09-proivder-transformation-functions.md]: Added in d6fb9c6.

File: _docs/usage/provider.md (hunk 1 above)
Comment (rgwozdz) [reply _docs/usage/provider.md]: Removed in d6fb9c6.

**Pull request merged**